libs/wire-api/src/Wire/API/Error/Galley.hs

```diff
@@ -206,6 +209,12 @@ type instance MapError 'MLSWelcomeMismatch = 'StaticError 400 "mls-welcome-misma
 
 type instance MapError 'MLSMissingGroupInfo = 'StaticError 404 "mls-missing-group-info" "The conversation has no group information"
 
+type instance MapError 'MLSMissingExternalInit = 'StaticError 400 "mls-missing-external-init" "The external commit is missing an external init proposal"
+
+type instance MapError 'MLSRemovalUserMismatch = 'StaticError 403 "mls-removal-user-mismatch" "The external commit attempts to remove a client from a user other than themselves"
+
+type instance MapError 'MLSExternalCommitRemoveProposal = 'StaticError 403 "mls-external-commit-proposals" "The external commit has to have at most one remove proposal"
```

Comment:
nitpick: "must have" instead of "has to have"

Reply:
I'll incorporate this only if there will be another reason to make another commit so as to avoid running CI again just for this nit-pick.

Reply:
I'll drop these errors in favor of the generic `MLSProtocolError`.